glambie/util/date_helpers.py

```diff
@@ -0,0 +1,85 @@
+from datetime import datetime, timedelta
+
+
+def decimal_dates_to_datetime_dates(decimal_dates_list: list) -> list:
+    """Function to convert a list of decimal dates to datetime dates
+
+    Parameters
+    ----------
+    decimal_dates_list : list
+        list of decimal date objects
+
+    Returns
+    ----------
+    A list of datetime dates
+    """
+    converted_dates = [year2datetime(i) for i in decimal_dates_list]
+    return converted_dates
+
+
+def datetime_dates_to_decimal_dates(datetime_dates_list: list) -> list:
+    """Function to convert a list of datetime dates to decimal dates
+
+    Parameters
+    ----------
+    datetime_dates_list : list
+        list of datetime date objects
+
+    Returns
+    ----------
+    A list of decimal dates
+    """
+    converted_dates = [datetime2year(i) for i in datetime_dates_list]
+    return converted_dates
+
+
+def datetime2year(datetime_date: datetime.date) -> float:
+    """Function to convert a datetime date to a decimal date
```

Comment:
rather than "decimal date", which [is misleading](https://en.wikipedia.org/wiki/Decimal_time), can we specifically say "fractional year"?

Reply:
done